src/robot/libdocpkg/xmlwriter.py

```diff
@@ -111,17 +128,3 @@ def _get_start_attrs(self, kw_type, kw, lib_source, writer):
     def _write_end(self, writer):
         writer.end('keywordspec')
         writer.close()
-
-
-class DocFormatter(object):
-
-    def __init__(self, doc_format, force_html=False):
-        if force_html:
-            self._formatter = DocToHtml(doc_format)
-            self.format = 'HTML'
-        else:
-            self._formatter = lambda doc: doc
-            self.format = doc_format
-
-    def __call__(self, doc):
-        return self._formatter(doc)
```

Comment:
DocFormatter could be completely removed due to new "builtIn" html converter in model